frontend/src/app/(navfooter)/new/NewScratchForm.tsx

```diff
@@ -18,7 +18,28 @@ import basicSetup from "@/lib/codemirror/basic-setup";
 import { cpp } from "@/lib/codemirror/cpp";
 import getTranslation from "@/lib/i18n/translate";
 
-import styles from "./new.module.scss";
+interface FormLabelProps {
+    children: React.ReactNode;
+    htmlFor?: string;
+    small?: string;
+}
+
+function FormLabel({ children, htmlFor, small }: FormLabelProps) {
```

Comment:
Call this `htmlEl` (i.e. element) rather than `htmlFor` because that has a very specific other meaning: https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor

```suggestion
function FormLabel({ children, htmlEl, small }: FormLabelProps) {
```